bundle/internal/tf/schema/data_source_serving_endpoints.go

```diff
@@ -3,7 +3,7 @@
 package schema
 
 type DataSourceServingEndpointsEndpointsAiGatewayGuardrailsInputPii struct {
-	Behavior string `json:"behavior"`
+	Behavior string `json:"behavior,omitempty"`
```

Comment:
Was this auto-generated?

Could you share a command to do this? (or link to doc if there is one).

Reply:
Yes, this is autogenerated from the TF provider schema.

Instructions (and code) at https://github.com/databricks/cli/tree/main/bundle/internal/tf/codegen

Reply:
Thanks, does this tool always fetch the most recent version?

Reply:
No, you have to manually change the version in this file before generating https://github.com/databricks/cli/blob/main/bundle/internal/tf/codegen/schema/version.go